postgis/postgis.sql.in

```diff
@@ -690,7 +690,7 @@ CREATE OR REPLACE FUNCTION geometry_overright(geom1 geometry, geom2 geometry)
 -- Availability: 0.1.0
 CREATE OPERATOR &> (
 	LEFTARG = geometry, RIGHTARG = geometry, PROCEDURE = geometry_overright,
-	COMMUTATOR = '&<',
+--	COMMUTATOR = '&<',
```

Comment:
@alesuiss this was fixed in trunk, use `git merge upstream/svn-trunk`

Reply:
Thanks :-)

We did install travis on our fork to test it there, but used PG 9.6 so didn't compile.